src/Feedback/index.js

```diff
@@ -235,20 +242,20 @@ const Feedback = ({
       ]}
       {...props}
     >
+      {side === "bottom" && (children
```

Comment:
You should use position absolute and transforms to correctly position the popover. Referring to open source Popover components will help.